docs/source/usage/network_scenarios.rst

```diff
@@ -0,0 +1,108 @@
+Network Scenarios
+=================
+
+Private Networks
+----------------
+
+If a lessee wishes to use a private network, they can run the following commands:
+
+  .. prompt:: bash $
+
+    openstack network create <network name>
+    openstack subnet create --subnet-range <subnet range> --allocation-pool start=<allocation start>,end=<allocation end> --network <network name> <subnet name>
+
+The created network will automatically have an assigned VLAN. This network can then be attached to a node as follows:
+
+  .. prompt:: bash $
+
+    openstack esi node network attach --network <network name> <node>
+
+In order for the lessee to access a node on a private network, they can use of the following options:
+
+* Contact the ESI administrator to connect the VLAN to an accessible node or VM outside of ESI
+* Use a trunk port whose native network is a VLAN that the lessee can already access - usually a public or external network - and which has a tagged network corresponding to the private network:
```

Comment:
I'm a little confused by the phrase, "...and which has a tagged network corresponding to the private network". Shouldn't the vlan be configured on that trunk by the following set of commands? The phrasing here suggests that the vlan was configured in advance.

Reply:
Ah, yeah, the intent is that this description is the outcome when following the commands beneath it. I've updated the wording a bit!